src/pymovements/dataset/dataset_files.py

```diff
@@ -492,9 +504,20 @@ def load_precomputed_event_file(
     if custom_read_kwargs is None:
         custom_read_kwargs = {}
 
-    valid_extensions = {'.csv', '.tsv', '.txt'}
-    if data_path.suffix in valid_extensions:
+    csv_extensions = {'.csv', '.tsv', '.txt'}
+    r_extensions = {'.rda'}
+    valid_extensions = csv_extensions | r_extensions
+    if data_path.suffix in csv_extensions:
         precomputed_event_df = pl.read_csv(data_path, **custom_read_kwargs)
+    elif data_path.suffix in r_extensions:
```

Comment:
please document this functionality in `Dataset.load_precomputed_reading_measures()`

Reply:
will add it after #1099 is merged due to merge conflicts